### PR TITLE
Added batch processing code for ReLu

### PR DESCRIPTION
**Pull request by KumareshN93**
* Verified for few matrix dimensions and batch sizes and the code works fine for all 15 ranks x arch. configs.

### CODE REVIEW

File: misc-bench/cpp-relu-batch/relu_batch.cpp [hunk 1]
```diff
@@ -0,0 +1,339 @@
...
+  // Define parameters for processing
+  PimDeviceProperties deviceProp;
+  PimStatus status = pimGetDeviceProperties(&deviceProp);
+  if (status != PIM_OK) {
+    std::cout << "Abort: pimGetDeviceProperties failed" << std::endl;
+    return 1;
+  }
+  // Get the device parameters
+  uint64_t numCols = uint64_t (deviceProp.numColPerSubarray);
+  uint64_t numRows = uint64_t (deviceProp.numRowPerSubarray);
```
Comment (fasiddique) [inline misc-bench/cpp-relu-batch/relu_batch.cpp:271]:
```suggestion
  uint64_t numCols = deviceProp.numColPerSubarray;
  uint64_t numRows = deviceProp.numRowPerSubarray;
```

File: misc-bench/cpp-relu-batch/relu_batch.cpp [hunk 2]
```diff
@@ -0,0 +1,339 @@
...
+  uint64_t numOfBits = uint64_t(deviceProp.numRanks) * uint64_t(deviceProp.numBankPerRank) * uint64_t (deviceProp.numSubarrayPerBank);
```
Comment (fasiddique) [inline misc-bench/cpp-relu-batch/relu_batch.cpp:272]: Shouldn't you multiply this wit rows and columns too?

Comment (KumareshN93) [reply misc-bench/cpp-relu-batch/relu_batch.cpp:272]: So, the multiplication of numofBits, numCols and numRows is done in the next step for numOfMatPerRow. Will update the variable names.

File: misc-bench/cpp-relu-batch/relu_batch.cpp [hunk 3]
```diff
@@ -0,0 +1,339 @@
...
+} Params;
+
+void usage()
+{
+  fprintf(stderr,
+          "\nUsage:  ./relu_batch.out [options]"
+          "\n"
+          "\n    -r    row (default=224)"
+          "\n    -c    column (default=224)"
+          "\n    -d    dimension (default=64)"
```
Comment (fasiddique) [inline misc-bench/cpp-relu-batch/relu_batch.cpp:32]:
```suggestion
          "\n    -r    row (default=226)"
          "\n    -c    column (default=226)"
          "\n    -d    dimension (default=128)"
```

**Review by fasiddique — APPROVED** [review]
Thank you! Can you please add the CPU and GPU implementation also?

**Pull request merged**